contracts/scripts/upgrade_placeholder.ts

```diff
@@ -0,0 +1,46 @@
+// run with: npx hardhat run <this file> --network local
+import dotenv from "dotenv";
+import { keccak256 } from "ethereumjs-util";
+import { ethers, upgrades } from "hardhat";
+
+dotenv.config({ path: __dirname + "/../.env" });
+
+const main = async () => {
+  const UUPSPlaceholderFactory = await ethers.getContractFactory(
+    "UUPSPlaceholder"
+  );
+  const V2Factory = await ethers.getContractFactory("UUPSPlaceholderV2");
```

Comment:
Don't use V2, just upgrade it to another placeholder implementation. this scripts is just for testing purpose. in real practice these placeholder should be upgraded to pre-deploys using deploy script under `deploy/l2`

Reply:
yeah sure, V2 was just for illustration / testing purposes. I removed it now